ui/helpers/utils/settings-search.test.js

```diff
@@ -111,10 +111,15 @@ const t = (key) => {
       return 'Localhost 8545';
     case 'experimental':
       return 'Experimental';
+    /** TODO: Remove during TOKEN_DETECTION_V2 feature flag clean up */
     case 'useTokenDetection':
       return 'Use Token Detection';
     case 'useTokenDetectionDescription':
       return 'We use third-party APIs to detect and display new tokens sent to your wallet. Turn off if you don’t want MetaMask to pull data from those services.';
+    case 'tokenDetection':
+      return 'Token Detection';
```

Comment:
the current translation in `app/_locales/en/messages.json` is actually "Token detection" with the lowercase "d". Not sure which one we should be using